docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -56,3 +56,28 @@ values will be skipped entirely.
 If there are missing values in feature fields (fields that are subjects of the 
 {dfanalytics}), then the document that contains the fields with the missing 
 values will be skipped during the analysis.
+
+[float]
+[[dfa-field-type-docs-limitations]]
+=== {dfanalytics-cap} field type and document limitation
```

Comment:
I'm not sure I understand why we need this section. Are there lots of other type/document limitations expected here? If not, I think it should be removed and the lower-level sections should be bumped up.